config/about.json

```diff
@@ -0,0 +1,11 @@
+{
+  "releases": [
+    {
+      "consoleVersion": "1.7.0",
+      "minimumCoreVersion": "3.0.0"
+    }, {
+      "consoleVersion": "1.6.0",
+      "minimumCoreVersion": "2.0.0"
+    }
+  ]
+}
```

Comment:
I think that Core shouldn't be in control of which version of console is compatible. This would mean that if changes has been made in console, that it will need to get an entry within the releases config of core as well. 

Ideally i'd think that OpenHIM console should send a request to core only requesting it for its current version. Then in the OpenHIM console config there should be a setting that specifies what the minimum allowed version of Core is that it will work with